### DISCUSSION TITLE
Wyze Home Monitoring

### DISCUSSION THREAD

**Discussion by IIAIronWolf** [SecKatie/ha-wyzeapi#376] [Ideas]
Any chance the newer Wyze Home Monitoring system will be supported here? Would like to integrate my security alarm status in HA.

**Comment by premanandc** [comment]
This was working until very recently. Did this get pulled because of a change on the Wyze side?

**Reply by IIAIronWolf** [reply]
To be honest, I haven't tried adding this integration again, because it didn't offer anything I wanted in the past. And it's not listed on the readme as supported.

**Reply by premanandc** [reply]
It is working. My Wyze hub lost connection and it became "unconfigured" in the app. So it wasn't showing up. I'll close this.

**Reply by premanandc** [reply]
I guess you'll have to do that.

**Comment by fleshymidget** [comment]
Currently the integration allows you to understand the status of the system overall (Armed/Disarmed etc). However we cant currently access the status of the sensors associated with the home monitoring system.  Is that something we could evaluate with some sort of MIM tool to understand the calls being made in the default wyze app?  Is there instructions anywhere for how to do something like that ?

**Reply by blachole** [reply]
Agreed, looking for this as well. Homebridge has it working, so its available in the API. I tried to review to see how easy it could be added compared to the HB code, but I am having some trouble.

**Reply by abhinav91690** [reply]
Looks like this should be a feature request

**Comment by N3rd7** [comment]
Can we actually change the status of the home monitoring through HA? So far, install went great and I see the home monitoring, but I can<t change the status from HA
